source/main/trezor/connection.ts

```diff
@@ -7,14 +7,25 @@ export const initTrezorConnect = async () => {
     await TrezorConnect.init({
       popup: false, // render your own UI
       webusb: false, // webusb is not supported in electron
-      debug: true,
+      debug: process.env.DEBUG_TREZOR === 'true',
```

Comment:
Since 8.2.8, `trezor-connect` actually implemented support for this parameter and started producing a huge amount of logs in the console. Generally those outputs seem useless but I've decided to left an option to turn them off for troubleshooting.